tests/magma_linalg_test.py

```diff
@@ -114,5 +114,36 @@ def testEigMagmaConfig(self):
       hlo = jax.jit(partial(lax_linalg.eig, use_magma=True)).lower(a).as_text()
       self.assertIn('magma = "on"', hlo)
 
+  @jtu.sample_product(
+    shape=[(3, 4), (3, 3), (4, 3), (4, 3)],
+    dtype=float_types + complex_types,
+  )
+  @jtu.run_on_devices("gpu")
+  def testPivotedQrFactorization(self, shape, dtype):
+    if jtu.jaxlib_version() <= (0, 5, 0):
+      self.skipTest("qr with `pivoting=True` on GPU requires jaxlib version > 0.4.38")
```

Comment:
nit: here and below, this version needs to be updated too :D

Reply:
Oops... Should have spotted that, thanks!